defaults/main.yml

```diff
@@ -15,6 +15,7 @@ clone_vm: DebianBusterTemplate    # The VM to clone
 clone_storage: local-lvm          # Target storage for full clone.
 clone_format: raw                 # Target drive's backing file's data format. Used only with clone
 clone_target: "{{ node }}"        # Target node 
+clone_vmid: 1                     # VM source id
```

Comment:
All right. Maybe it's a good time to also assign a var to the `newid` argument
```yaml
clone_newid: 9
```

Reply:
I agree, regardless of the documentation, are you sure if this parameter works? does it work to indicate the new LXC id to the cluster?

Reply:
I've tested the `newid` parameter and it does work.
I tested the `vmid` parameter this morning too.